source/edge-server/manage-edge-servers.txt

```diff
@@ -90,29 +88,42 @@ Services.
 
       .. include:: /includes/create-service-data-services.rst
 
-      .. include:: /includes/edge-server-enable-device-sync.rst
-
    .. tab:: Create an App Service
       :tabid: app-service
 
       Refer to the documentation to :ref:`create-app`.
 
-      When you choose your App type, select ``Edge Server`` or ``Device Sync``. 
+      When you choose your App type, select ``Device Sync``. You can add
+      Edge Server to your Device Sync App after you configure Device Sync.
 
-      .. include:: /includes/edge-server-enable-device-sync.rst
+      [PLACEHOLDER: Add steps for adding Edge Server to a Device Sync App.
+      It's not 100% clear to me what the flow is from the Figma designs.
+      Populate the rest of this procedure when we are able to test this flow.]
 
 Supported Authentication Providers
 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
-Edge Server supports a subset of authentication providers:
+Edge Server supports a subset of authentication providers. Supported 
+authentication providers vary depending on whether you're connecting with
+Atlas Device SDK or using a MongoDB URI to connect with a MongoDB Driver 
+or tool.
+
+You must enable at least one of these providers to enable clients to connect
+to the Edge Server.
+
+Supported Atlas Device SDK Authentication Providers
+```````````````````````````````````````````````````
 
 - :ref:`anonymous-authentication`
 - :ref:`email-password-authentication`
 - :ref:`custom-jwt-authentication`
 - :ref:`api-key-authentication`
 
-You must enable at least one of these providers to enable clients to connect
-to the Edge Server.
+Supported MongoDB URI Authentication Providers
+``````````````````````````````````````````````
+
+- :ref:`anonymous-authentication`
```

Comment:
@sudssm Hoping to get your input here, do we want to represent the --insecure-disable-auth flow like this here? Or do you think we should drop this part?

Reply:
I'm going to go ahead and merge this PR now to keep things moving along, but I'm happy to change this in the feature branch before publication if there's a different way you'd prefer to communicate these details.